3422.xml

```diff
@@ -0,0 +1,502 @@
+				<Type>Float</Type>
+				<RangeEnumeration/>
+				<Units>varh</Units>
+				<Description><![CDATA[Reactive energy import, unified rate, on the 3 lines (+R).]]></Description>
+			</Item>
+			<Item ID="22">
+				<Name>Reactive energy export Un</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration/>
+				<Units>varh</Units>
+				<Description><![CDATA[Reactive energy export, unified rate, on the 3 lines (-R).]]></Description>
+			</Item>
+			<Item ID="23">
+				<Name>Reactive energy QI Un</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration/>
+				<Units>varh</Units>
+				<Description><![CDATA[Reactive energy QI, unified rate, on the 3 lines (+Ri).]]></Description>
+			</Item>
+			<Item ID="24">
+				<Name>Reactive energy QII Un</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration/>
+				<Units>varh</Units>
+				<Description><![CDATA[Reactive energy QII, unified rate, on the 3 lines (+Rc).]]></Description>
+			</Item>
+			<Item ID="25">
+				<Name>Reactive energy QIII Un</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration/>
+				<Units>varh</Units>
+				<Description><![CDATA[Reactive energy QIII, unified rate, on the 3 lines (-Ri).]]></Description>
+			</Item>
+			<Item ID="26">
+				<Name>Reactive energy QIV Un</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration/>
+				<Units>varh</Units>
+				<Description><![CDATA[Reactive energy QIV, unified rate, on the 3 lines (-Rc).]]></Description>
+			</Item>
+			<Item ID="27">
+				<Name>Apparent energy import Un</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration/>
+				<Units>VAh</Units>
+				<Description><![CDATA[Apparent energy import, unified rate, on the 3 lines.]]></Description>
+			</Item>
+			<Item ID="28">
+				<Name>Apparent energy export Un</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration/>
+				<Units>VAh</Units>
+				<Description><![CDATA[Apparent energy export, unified rate, on the 3 lines.]]></Description>
+			</Item>
+			<Item ID="29">
+				<Name>Number of power failures in any phase</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
```

Comment:
Can the number of failures be a non-integer value? Should this be an unsigned integer instead of float?

Reply:
Thank you Ari for pointing out.
You are perfectly right.
Changing to Unsigned Integer.